vorestation.dme

```diff
@@ -1,4065 +1,4066 @@
-#include "code\modules\xenobio\machinery\processor.dm"
-#include "code\modules\xgm\xgm_gas_data.dm"
-#include "code\modules\xgm\xgm_gas_mixture.dm"
-#include "code\unit_tests\decl_tests.dm"
-#include "code\unit_tests\language_tests.dm"
-#include "code\unit_tests\loadout_tests.dm"
-#include "code\unit_tests\map_tests.dm"
-#include "code\unit_tests\material_tests.dm"
-#include "code\unit_tests\mob_tests.dm"
-#include "code\unit_tests\recipe_tests.dm"
-#include "code\unit_tests\research_tests.dm"
-#include "code\unit_tests\sqlite_tests.dm"
-#include "code\unit_tests\subsystem_tests.dm"
-#include "code\unit_tests\unit_test.dm"
-#include "code\unit_tests\unit_test_vr.dm"
-#include "code\unit_tests\vore_tests_vr.dm"
-#include "code\unit_tests\zas_tests.dm"
-#include "code\unit_tests\integrated_circuits\arithmetic.dm"
-#include "code\unit_tests\integrated_circuits\circuits.dm"
-#include "code\unit_tests\integrated_circuits\converter.dm"
-#include "code\unit_tests\integrated_circuits\logic.dm"
-#include "code\unit_tests\integrated_circuits\trig.dm"
-#include "code\ZAS\Airflow.dm"
-#include "code\ZAS\Atom.dm"
-#include "code\ZAS\Connection.dm"
-#include "code\ZAS\ConnectionGroup.dm"
-#include "code\ZAS\ConnectionManager.dm"
-#include "code\ZAS\Controller.dm"
-#include "code\ZAS\Debug.dm"
-#include "code\ZAS\Diagnostic.dm"
-#include "code\ZAS\Fire.dm"
-#include "code\ZAS\Phoron.dm"
-#include "code\ZAS\Turf.dm"
-#include "code\ZAS\Variable Settings.dm"
-#include "code\ZAS\Zone.dm"
-#include "interface\interface.dm"
-#include "interface\skin.dmf"
-#include "maps\gateway_archive_vr\blackmarketpackers.dm"
-#include "maps\offmap_vr\om_ships\abductor.dm"
-#include "maps\southern_cross\items\clothing\sc_accessory.dm"
-#include "maps\southern_cross\items\clothing\sc_suit.dm"
-#include "maps\southern_cross\items\clothing\sc_under.dm"
-#include "maps\southern_cross\loadout\loadout_suit.dm"
-#include "maps\southern_cross\loadout\loadout_uniform.dm"
-#include "maps\southern_cross\loadout\loadout_vr.dm"
-#include "maps\submaps\_helpers.dm"
-#include "maps\submaps\_readme.dm"
-#include "maps\submaps\engine_submaps\engine.dm"
-#include "maps\submaps\engine_submaps\engine_areas.dm"
-#include "maps\submaps\engine_submaps_vr\tether\_engine_submaps.dm"
-#include "maps\submaps\space_submaps\space.dm"
-#include "maps\submaps\space_submaps\debrisfield\debrisfield.dm"
-#include "maps\submaps\surface_submaps\mountains\mountains.dm"
-#include "maps\submaps\surface_submaps\mountains\mountains_areas.dm"
-#include "maps\submaps\surface_submaps\mountains\mountains_areas_vr.dm"
-#include "maps\submaps\surface_submaps\plains\plains.dm"
-#include "maps\submaps\surface_submaps\plains\plains_areas.dm"
-#include "maps\submaps\surface_submaps\wilderness\wilderness.dm"
-#include "maps\submaps\surface_submaps\wilderness\wilderness_areas.dm"
-#include "maps\tether\tether.dm"
-#include "maps\~map_system\maps.dm"
-// END_INCLUDE
+// DM Environment file for baystation12.dme.

```

Comment:
Your PR changes all the line endings in vorestation.dme to LF instead of the proper-for-this-repo CRLF.

Reply:
Ok how do I accept your changes? Or is this a review that has changes in it?

Reply:
It doesn't have changes in it, I can't do a suggestion that's 4000+ lines long. :P

Reply:
Then how do I fix it? I legit just downloaded and compiled it off a fresh copy of the (at the time) latest version of the server?

Reply:
I touched nothing else.

Reply:
Your editor touched something else. You'll have to use a program like vscode or notepad++ or something that can change line endings